docs/reference/organizations-on-nuget-org.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Organizations on nuget.org | Microsoft Docs
+author: anangaur
+ms.author: anangaur
+manager: unnir
+ms.date: 04/10/2018
+ms.topic: article
+ms.prod: nuget
+ms.technology: null
+description: Organizations on nuget.org helps you to manage packages published by group or in a team, company environment.
+ms.reviewer:
+- karann-msft
+- kraigb
+- camsoper
+---
+
+# Organization on nuget.org
+
+Organizations enables businesses and open-source projects to collaborate across many packages at once using a single nuget.org identity. For a package consumer, an organization account appears same as an existing user account on nuget.org.
+
+## User accounts vs. organization accounts
+
+Your user account is your identity on nuget.org and can be a member of any number of organizations. A package can belong to an organization account like it can belong to a user account. Package consumers don't see any difference between an user account or the organization account: both appear as package `owners`.
+
+An organization account has one or more user accounts as its members. These members can manage a set of packages while maintaining a single identity for ownership.
+
+## Adding a new organization
+
+To add a new organization, select your account on nuget.org, then select the **Manage Organizations...** menu command:
+
+![Menu option on nuget.org for Manager Organizations](media/org-manage-option.png)
+
+On the next page, select the **Add new organization** button:
+
+![Button to create a new organization on nuget.org](media/org-add-new-option.png)
+
+On the next page, provide the organization name and email address. Because nuget.org manages organization and user names together, the organization name must be different from any other existing organization or user accounts. The email address must also be unique across all accounts.
```

Comment:
Remove " Because nuget.org manages organization and user names together"
It's an implementation detail. The reason to not allow collisions between users and orgs is to avoid confusion - both can own packages.

Reply:
Makes sense.